Client/src/Components/ListRouter.tsx

```diff
@@ -21,8 +21,8 @@ import { ResearchPaperAnalysisView } from "./ResearchPaperAnalysis/ResearchPaper
 import { Route } from 'react-router'
 import SearchIcon from '@material-ui/icons/Search'
 import SearchView from "./Search/SearchView"
-import SignInView from "../Components/Authentication/SignInView"
-import SignUpView from "../Components/Authentication/SignUpView"
+import SignInView from "./Authentication/SignInView"
```

Comment:
Can we rename this to login instead of signin

Reply:
yep agree with this

Reply:
renamed it everywhere

Reply:
renamed everywhere